### PR TITLE
Paypal smart button exception trans corrections.

### PR DESCRIPTION
**Pull request by papnoisanjeev**

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
**Not foxed,** please find the **screenshot** [Here](https://prnt.sc/nXtY3YOC9tVP)

**Comment by papnoisanjeev** [comment]
@Sanjaybhattwebkul 

Check now.

File: packages/Webkul/Shop/src/Resources/lang/hi_IN/app.php [hunk 1]
```diff
@@ -544,6 +544,7 @@
             'specify-shipping-method'   => 'कृपया शिपिंग विधि निर्दिष्ट करें।',
             'specify-payment-method'    => 'कृपया भुगतान विधि निर्दिष्ट करें।',
             'rule-applied'              => 'Cart rule applied',
+            'paypal-payment-canceled'   => 'Paypal payment has been canceled.',
```
Comment (Sanjaybhattwebkul) [inline packages/Webkul/Shop/src/Resources/lang/hi_IN/app.php:547]: Please add translation here

File: packages/Webkul/Shop/src/Resources/lang/hi_IN/app.php [hunk 2]
```diff
@@ -544,6 +544,7 @@
             'specify-shipping-method'   => 'कृपया शिपिंग विधि निर्दिष्ट करें।',
             'specify-payment-method'    => 'कृपया भुगतान विधि निर्दिष्ट करें।',
             'rule-applied'              => 'Cart rule applied',
```
Comment (Sanjaybhattwebkul) [inline packages/Webkul/Shop/src/Resources/lang/hi_IN/app.php:546]: Add translation

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
Now this is **working fine**, Please find the **screenshot** [Here](https://prnt.sc/xgUJx0P2Layw)

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**